app/js/services/resultatService.js

```diff
@@ -2,20 +2,25 @@
 
 angular.module('ddsApp').service('ResultatService', function($http, $modal, droitsDescription) {
 
+    var DROITS_POTENTIELS = droitsDescription.prestationsNationales.concat(
+        _.flatten(_.pluck(droitsDescription.partenairesLocaux, 'prestations'))
```

Comment:
Yes, sorry  :wink:

> Is `Array.prototype.concat.apply` the same than `[].concat.apply` ?

Same effect, but not exactly the same thing happens. In the latter case, you instantiate a new `Array` to get access to its `concat` method, while in the former you just access to it straight to the already-instantiated `prototype` reference. Prefer the former for performance.